app/src/main/assets/shared.css

```diff
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: 'Material Icons';
+  font-style: normal;
+  font-weight: 400;
+  src: url(file:///android_res/raw/materialicons_regular.ttf) format('truetype');
```

Comment:
I would prefer to move the file to `assets` and simply use `url('materialicons_regular.ttf')` here.